crates/examples/attestation/README.md

```diff
@@ -1,75 +1,109 @@
 ```
 
-If successful, you should see this output in the console:
+If successful, you’ll see this output in the console:
 
 ```log
 Presentation built successfully!
-The presentation has been written to `example.presentation.tlsn`.
+The presentation has been written to `example-json.presentation.tlsn`.
 ```
 
-### 3. Verify the presentation
+You can create multiple presentations from the attestation and secrets in the notarization step, each with customized data redactions. You are invited to experiment!
 
-This will read the presentation from the previous step, verify it, and print the disclosed data to console.
+### 3. Verify the Presentation
 
-Run the `verify` binary.
+This step reads the presentation created above, verifies it, and prints the disclosed data to the console.
 
+Run the verify binary:
 ```shell
 cargo run --release --example attestation_verify
 ```
 
-If successful, you should see this output in the console:
-
+Upon success, you should see output similar to:
 ```log
 Verifying presentation with {key algorithm} key: { hex encoded key }
 
 **Ask yourself, do you trust this key?**
 
 -------------------------------------------------------------------
-Successfully verified that the data below came from a session with example.com at 2024-10-03 03:01:40 UTC.
+Successfully verified that the data below came from a session with test-server.io at 2024-10-03 03:01:40 UTC.
 Note that the data which the Prover chose not to disclose are shown as X.
 
 Data sent:
 ...
 ```
 
-⚠️ Notice that the presentation comes with a "verifying key". This is the key the Notary used when issuing the attestation that the presentation was built from. If you trust the Notary, or more specifically the verifying key, then you can trust that the presented data is authentic.
+⚠️ The presentation includes a “verifying key,” which the Notary used when issuing the attestation. If you trust this key, you can trust the authenticity of the presented data.
 
-### Next steps
+### HTML
 
-Try out the [Discord example](../discord/README.md) and notarize a Discord conversations.
+In the example above, we notarized a JSON response. TLSNotary also supports notarizing HTML content. To run an HTML example, use:
 
+```shell
+# notarize
+SERVER_PORT=4000 cargo run --release --example attestation_prove -- html
+# present
+cargo run --release --example attestation_present -- html
+# verify
+cargo run --release --example attestation_verify -- html
+```
 
+### Private Data
+
+The examples above demonstrate TLSNotary with public data. TLSNotary can also be used for private data by adding the necessary headers (e.g., an authentication token) or cookies to the request. To run a private data example:
+```shell
+# notarize
+SERVER_PORT=4000 cargo run --release --example attestation_prove -- authenticated
+# present
+cargo run --release --example attestation_present -- authenticated
+# verify
+cargo run --release --example attestation_verify -- authenticated
+```
```

Comment:
In the first example there are already redacted passages, which I would consider private data? My understanding is that you want to emphasize that the redaction feature can be used to hide authentication secrets like passwords, cookies. In this case, I would make it more clear here.

Reply:
The difference here is that you need an authentication token to access the data. I'll try to explain it better